pkg/middleware/region_proxy.go

```diff
@@ -6,33 +6,64 @@ import (
 	"net/http"
 	"net/http/httputil"
 	"net/url"
+	"strings"
+	"time"
 
+	"github.com/hashicorp/golang-lru/v2/expirable"
 	sdk "github.com/openshift-online/ocm-sdk-go"
 	"github.com/openshift-online/ocm-sdk-go/logging"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const (
+	// region ids
+	APSoutEast1 = "ap-southeast-1"
+	USEast1     = "us-east-1"
```

Comment:
These could be defined in tests only it seems

Reply:
why do we need the keys hard-coded in this repo at all?

Reply:
@markturansky 
are the keys defined in some common repo? 
If yes, we can use that. 
I have not found that, and in osl, the rhregionid is also hardcoded.

Reply:
I don't think there is a need for this ID in common at all.   Whatever rh_region_id is returned from AMS gets looked up in ocm-shards.json.  The key is irrelevant to ocm-common.